cinema-web/src/main/resources/application-local.properties

```diff
@@ -0,0 +1,13 @@
+spring.jpa.properties.jakarta.persistence.sharedCache.mode=ENABLE_SELECTIVE
+spring.jpa.open-in-view=false
+spring.jpa.properties.hibernate.format_sql=true
+spring.jpa.properties.hibernate.use_sql_comments=true
+server.port=9090
+# H2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2
+# JPA
+spring.jpa.show-sql=true
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
Czy to jest dobre zachowanie dla środowiska produkcyjnego?

Reply:
Nie jest. Ekspozycja h2 + jak już pokazywać sql to lepiej do jakiś logów aby się zapisywały do pliku niż zaśmiecać konsolę. Ale dlatego też dałem application-local. Tylko do testów małych używałem tego :)